### ISSUE TITLE
[A11y] Add border for header

### ISSUE THREAD

**Issue by liviakuenzli** [lyne-design-system/lyne-components#1592]
## Detailed description

According to our first accessibility review by Mürsel Genc, the header and footer are hard to recognize because they are not contrasted from the background.
In high contrast mode (Windows) sections like header or footer should be given a border by CSS. Non-visible attributes, e.g. a transparent border color, can be used here, as this then becomes visible when the contrast is adjusted.

![image](https://user-images.githubusercontent.com/7348876/216984479-8eb21e2c-a49b-4681-91cf-90c09365081f.png)

See also: https://flow.sbb.ch/browse/SBBCH-15182
